### ISSUE TITLE
User data directory location

### ISSUE THREAD

**Issue by hlolli** [mb21/panwriter#17]
Hi and thanks for this pandoc editor. I'm hoping to recommend it for a new project. I would be planing to use this as a tool for previewing, but not rendering, which would take place on a CI server. I notice that it's possible to set default.yml in a so called user data directory. But what I'd really want, is to have a configuration set from the same location, that way the respository we'd be working with would be self contained and every contributor would be editing the same .css files using the same configuration. Would it be possible for panwriter to read configuration files from the same directory as the source file? In the end, relative paths to .css files is what I'm mostly after, and since we have so many files, it wouldn't make sense to copy paste the styles into the metadata section of each file.

**Comment by mb21** [comment]
Yes, that would certainly make sense! Would you like to make a pull request? I might be able to implement it myself, but can't promise how soon...

Are you planning on doing the server-side rendering using pagedjs or using pandoc (which pdf-engine)?
